services/course.js

```diff
@@ -4,11 +4,11 @@ const { DOCUMENT_NOT_FOUND } = require('~/consts/errors')
 const { createError, createForbiddenError } = require('~/utils/errorsHelper')
 
 const courseService = {
-  getCourses: async ({ author, skip, limit }) => {
+  getCourses: async ({ author, skip, limit, sort }) => {
     const items = await Course.find({ author })
       .skip(skip)
       .limit(limit)
-      .sort({ updatedAt: -1 })
+      .sort({ updatedAt: sort })

```

Comment:
maybe better to pass 'updatedAt asc' in sort value, in this case this sort can work for any field, but we need to think about it